relay/sources/rudder-package/src/cli.rs

```diff
@@ -15,4 +15,19 @@ pub struct Args {
     /// Enable verbose logs
     #[arg(short, long, default_value_t = false)]
     pub debug: bool,
+
+    #[clap(subcommand)]
+    pub command: Command,
+}
+
+#[derive(Debug, Subcommand)]
+pub enum Command {
+    Install {
+        #[clap(long, short = 'f', help = "Force installation of given plugin")]
+        force: bool,
+
+        #[clap()]
+        package: Option<String>,
```

Comment:
why an Option?

Reply:
Because of the `--list` option that's been removed. Will fix